editor/src/uuiui/tooltip.tsx

```diff
@@ -33,41 +22,24 @@ export class Tooltip extends React.Component<React.PropsWithChildren<TooltipProp
   }
 
   render() {
-    return (
-      <Tippy
-        css={{
-          fontWeight: 400,
-          fontSize: 11,
-          fontFamily:
-            "utopian-inter, -apple-system, BlinkMacSystemFont, Helvetica, 'Segoe UI', Roboto, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'",
-          backgroundColor: `${colorTheme.neutralInvertedBackground.value} !important`,
-          color: `${colorTheme.neutralInvertedForeground.value} !important`,
-          '& .tippy-content': {
-            padding: '4px 8px !important',
-          },
-          '&[data-placement^=top] .tippy-arrow::before': {
-            borderTopColor: `${colorTheme.neutralInvertedBackground.value} !important`,
-          },
-          '&[data-placement^=right] .tippy-arrow::before': {
-            borderRightColor: `${colorTheme.neutralInvertedBackground.value} !important`,
-          },
-          '&[data-placement^=bottom] .tippy-arrow::before': {
-            borderBottomColor: `${colorTheme.neutralInvertedBackground.value} !important`,
-          },
-          '&[data-placement^=left] .tippy-arrow::before': {
-            borderLeftColor: `${colorTheme.neutralInvertedBackground.value} !important`,
-          },
-        }}
-        arrow
-        disabled={this.props.disabled}
-        content={this.props.title}
-        placement={this.props.placement ?? 'top'}
-        delay={[500, 0]} // [show, hide] milliseconds
-        animation='fade'
-        theme='material'
-      >
-        {this.props.children}
-      </Tippy>
-    )
+    if (this.props.disabled) {
+      return this.props.children
+    } else {
+      return (
+        <div
+          css={{
+            position: 'relative',
+            '&:hover:after': {
+              background: 'orange',
```

Comment:
something tells me this is a placeholder design

Reply:
It's the one I was supplied with!

Reply:
Yeah, this was the sample code to even test if it solves the re-render problem and still even _displays_, what with z-index issues lurking.